src/main.rs

```diff
@@ -124,6 +127,12 @@ fn find_ck3_dir_in_steam_dir(steam_dir: PathBuf) -> Option<PathBuf> {
 fn main() -> Result<()> {
     let mut args = Cli::parse();
 
+    #[cfg(windows)]
+    if !args.no_color {
+        let _ = ansi_term::enable_ansi_support()
+            .map_err(|_| eprintln!("Failed to run enable_ansi_support() for Windows10 users. Continuing probably without colored output."));
```

Comment:
Since this is user-facing it should probably just start with "Failed to enable ANSI support ..." instead of naming the function.